### ISSUE TITLE
Linking Skills is hampered by selector when one of the skill is an prefix of another skill

### ISSUE THREAD

**Issue by fabianhjr** [talenet/talenet#29]
Hello, when linking skills for instance "statistics" and "math", "Mathematica" gets in the way of the selector.

![screenshot from 2018-03-01 21-39-51](https://user-images.githubusercontent.com/303897/36882402-3ce58a12-1d99-11e8-9fdc-86de7ead7c26.png)

**Comment by cryptix** [comment]
Thank you for reporting This! I can see this being quite annoying.

I'm not sure about the best way forward. I imagine something like a dropdown under the text entry could work much better to pick ambiguities here but we are always open for better suggestions.

**Comment by fabianhjr** [comment]
Yeah, sure thing. Love the concept, gift / sharing talent platform for the commons.

Will try to give it a go soonish. ^_^
